erpnext/patches.txt

```diff
@@ -262,6 +262,7 @@ erpnext.patches.v14_0.update_reference_due_date_in_journal_entry
 erpnext.patches.v15_0.saudi_depreciation_warning
 erpnext.patches.v15_0.delete_saudi_doctypes
 erpnext.patches.v14_0.show_loan_management_deprecation_warning
+execute:frappe.rename_doc("Report", "TDS Payable Monthly", "Tax Withholding Details", force=True)
```

Comment:
Can we move this to `[post_model_sync]`? Seems to break my migration otherwise (`rename_doc` triggers a query on `tabDocType.is_virtual`, which does not exists at that time).